vue/sbc-common-components/src/services/token.services.ts

```diff
@@ -0,0 +1,119 @@
+import Keycloak, { KeycloakInitOptions, KeycloakInstance } from 'keycloak-js'
+
+class TokenServices {
+  private kc: KeycloakInstance
+  private static instance: TokenServices
+  private kcLoaded
+  private counter = 0
+  private timerId
+  private kcOptions: KeycloakInitOptions = {
+    onLoad: 'login-required',
+    checkLoginIframe: false,
+    timeSkew: 0,
+    token: sessionStorage.getItem('KEYCLOAK_TOKEN'),
+    refreshToken: sessionStorage.getItem('KEYCLOAK_REFRESH_TOKEN'),
+    idToken: sessionStorage.getItem('KEYCLOAK_ID_TOKEN')
+  }
+
+  initUsingKc (kcInstance: KeycloakInstance) {
+    console.info('[TokenServices] KC Instance registered using kc instance')
```

Comment:
How about providing an option to disable all info messages? This module is very noisy :)

Reply:
Sure Thing.I will add it